packages/process/test/exec.test.ts

```diff
@@ -158,26 +181,166 @@ describe('exec', () => {
   // in windows at all.
   if (process.platform !== 'win32') {
     describe('when the `shell` option is true', () => {
-      it('lets the shell do all of the shellword parsing', function*() {
+      it('lets the shell do all of the shellword parsing', function* () {
         let proc = exec('echo "first" | echo "second"', {
-          shell: true
+          shell: true,
         });
         let { stdout }: ProcessResult = yield proc.expect();
 
-        expect(stdout).toEqual("second\n");
-      })
+        expect(stdout).toEqual('second\n');
+      });
     });
   }
 
   describe('when the `shell` option is `false`', () => {
-    it('automatically parses the command argumens using shellwords', function*() {
+    it('automatically parses the command arguments using shellwords', function* () {
       let proc = exec('echo "first" | echo "second"', {
-        shell: false
+        shell: false,
       });
       let { stdout }: ProcessResult = yield proc.expect();
 
-      expect(stdout).toEqual("first | echo second\n");
-    })
+      expect(stdout).toEqual('first | echo second\n');
```

Comment:
It looks like there are a bunch of un-related formatting changes in this PR. If we want to add automated code-formatters, that should be done as part of a separate PR.